helpers/mathjax.js

```diff
@@ -61,35 +61,19 @@ export class HtmlBlockMathRenderer {
 		// This work-around should be removed when linebreaks are natively supported.
 		// MathJax issue: https://github.com/mathjax/MathJax/issues/2312
 		// A duplicate that explains our exact issue: https://github.com/mathjax/MathJax/issues/2495
-		const lineBreakStyle = 'display: block; height: 0.5rem;';
-
-		// If we're opting out of deferred rendering, we need to rely
-		// on the global MathJax install for rendering.
-		if (options.noDeferredRendering) {
-			elem.querySelectorAll('mspace[linebreak="newline"]').forEach(elm => {
-				elm.setAttribute('style', lineBreakStyle);
-			});
-
-			await window.MathJax.startup.promise;
-			renderingPromise = renderingPromise.then(() => window.MathJax.typesetShadow(elem.getRootNode(), elem));
-			await renderingPromise;
-			return elem;
-		}
-
-		const inner = elem.innerHTML.replace(/<mspace linebreak="newline">/gi, `<mspace linebreak="newline" style="${lineBreakStyle}">`);
-
-		const temp = document.createElement('div');
-		temp.style.display = 'none';
-		temp.attachShadow({ mode: 'open' });
-		temp.shadowRoot.innerHTML = `<div><mjx-doc><mjx-head></mjx-head><mjx-body>${inner}</mjx-body></mjx-doc></div>`;
+		elem.querySelectorAll('mspace[linebreak="newline"]').forEach(elm => {
+			elm.setAttribute('style', 'display: block; height: 0.5rem;');
```

Comment:
I know this is what we were already doing, but is there a chance that we're blowing away properties that already exist on `mspace`? If so, is it intentional, or should we be using `elm.style.display = 'block';` etc.?

Reply:
`mspace` shouldn't have any additional properties according to spec, but I think your suggestion is safer regardless so I'll just go with that.

Reply:
If it's invalid otherwise then it's all good. If there is a chance to unexpected clobber, then we should play it safe.

Reply:
Yeah, it shouldn't be valid for there to be styling on there because MathJax will just wipe it out anyways, but it's a trivial change and strictly safer, so... 🤷